.travis.yml

```diff
@@ -42,11 +41,19 @@ before_install:
     ${run_opts} ${distribution}-${version}:ansible "${init}"
 
 install:
-  - 'pip install -r requirements-devel.txt'
+  - 'sudo apt-get update -qq'
+  - 'sudo apt-get install -y python-pip'
+  - 'sudo pip install -r requirements-devel.txt'
+  - 'sudo pip install ansible'
```

Comment:
Oh, yeah, about that, if we do `language: 'python'`, there will be a Python virtualenv already setup (with `pip` installed) ([Travis CI Uses Isolated virtualenvs](https://docs.travis-ci.com/user/languages/python#Travis-CI-Uses-Isolated-virtualenvs)). Could you check if that's the case?